### DISCUSSION TITLE
Disable experimental OT warning

### DISCUSSION THREAD

**Discussion by romanzdk** [apache/airflow#40218] [Q&A]
How to disable annoying warning about Open Telemetry being experimental, please?

![image](https://github.com/apache/airflow/assets/31843161/41d41409-2c20-4ce1-92bd-f007283b135e)

**Comment by potiuk** [comment]
I don't think it's possible (@ferruzzi?) - maybe we should disable the warning altogether and cherry-pick that change to next patchlevel ? It might be indeed annoying

**Comment by ferruzzi** [comment]
Yeah, I've been thinking it's safe to remove that banner.  I don't think there have been any big bugs reported and I think it's safe to remove.

**Comment by ferruzzi** [comment]
This PR should remove the banner for Metrics, but it will still be there if you use the newly-added Traces.

https://github.com/apache/airflow/pull/40286

**Comment by potiuk** [comment]
I think the banner was (in a hindsight) a bad idea, log entry should be enough. Possibly we should remove it for both (separate PR for traces so that we can cherry-pick #40286 to 2.9.3)
